src/features/surveys/components/SurveySubmissionsList.tsx

```diff
@@ -157,20 +161,28 @@ const SurveySubmissionsList = ({
     }
   };
 
-  const EditCell: FC<{ row: ZetkinSurveySubmission }> = ({ row }) => {
+  const EditCell: FC<{
+    respondents: ZetkinPerson[];
+    row: ZetkinSurveySubmission;
+  }> = ({ respondents, row }) => {
     const api = useGridApiContext();
     const { orgId } = useRouter().query;
-    const email = row.respondent?.email || '';
+
+    const emailOrName =
+      row.respondent?.email ||
+      row.respondent?.first_name ||
+      row.respondent?.last_name ||
+      '';
     let { data: suggestedPeople } = useQuery(
-      ['peopleSearchResults', email],
-      getPeopleSearchResults(email, orgId as string),
+      ['peopleSearchResults', emailOrName],
+      getPeopleSearchResults(emailOrName, orgId as string),
       {
-        enabled: email.length >= 2,
+        enabled: emailOrName.length >= 2,
         retry: true,
       }
     );
 
-    if (!suggestedPeople) {
+    if (!suggestedPeople || suggestedPeople === undefined) {
       suggestedPeople = [];
```

Comment:
isn't `undefined` falsy? I assume there is a reason for this addition but it made me curious.